node/src/server-modules/GlideJson.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+import { DecoderOption, GlideString } from "../BaseClient";
+import { ConditionalChange } from "../Commands";
+import { GlideClient } from "../GlideClient";
+import { GlideClusterClient } from "../GlideClusterClient";
+
+export type ReturnTypeJson = GlideString | GlideString[];
+/**
+ * Represents options for formatting JSON data, to be used in the [JSON.GET](https://valkey.io/commands/json.get/) command.
+ */
+export interface JsonGetOptions {
+    /** The path or list of paths within the JSON document. Default is root `$`. */
+    paths?: GlideString[];
+    /** Sets an indentation string for nested levels. Defaults to None. */
+    indent?: GlideString;
+    /** Sets a string that's printed at the end of each line. Defaults to None. */
+    newline?: GlideString;
+    /** Sets a string that's put between a key and a value. Defaults to None. */
+    space?: GlideString;
+}
```

Comment:
`NOESCAPE` is missing

Reply:
this option is also not present in python, does this mean it also need updating?

Reply:
I think so (unless it is a noop)